README.md

```diff
@@ -660,7 +660,12 @@ THREDDED_TESTAPP_WEBPACK=1 bin/rails s
 
 ### Testing
 
-To run the tests, just run `rspec`. The test suite will re-create the test database on every run, so there is no need to
+In order to run the tests locally, you will need to be running webpack-dev-server (or do a manual compilation):
+
+    cd spec/dummy && yarn && cd -
+    BUNDLE_GEMFILE=$(realpath)/Gemfile spec/dummy/bin/webpack-dev-server
```

Comment:
Is `BUNDLE_GEMFILE` necessary?
If it is, best quote the path and no need to call `realpath`:

```bash
BUNDLE_GEMFILE="${PWD}/Gemfile"
```

Reply:
@glebm Yes:

     $ spec/dummy/bin/webpack-dev-server
    /Users/tim/workspaces/thredded/thredded/spec/dummy/Gemfile not found